Doc/library/subprocess.rst

```diff
@@ -356,6 +356,25 @@ functions.
    arguments for additional differences from the default behavior.  Unless
    otherwise stated, it is recommended to pass *args* as a sequence.
 
+   .. warning::
```

Comment:
Can you also change `os.execvp` to `os.execvpe` at the beginning of the `Popen` docs? 

Also, further down in the `cwd` section it claims "In particular, the function looks for *executable* (or for the first item in *args*) relative to *cwd* if the executable path is a relative path". That should be changed to restrict it to just POSIX. You could change "In particular" to "In POSIX".

Reply:
Done.